.github/workflows/pull-request.yml

```
@@ -30,6 +30,48 @@ jobs:
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v45
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.9
+
+      - name: Run Python script
+        run: |
+          python .github/workflows/scripts/code_coverage_disable_check.py --files ${{ steps.changed-files.outputs.all_changed_files }} 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)